core/domain/base_model_validators.py

```diff
@@ -307,14 +307,16 @@ def _validate_model_time_fields(cls, item):
         Args:
             item: ndb.Model. Entity to validate.
         """
-        if item.created_on > item.last_updated:
+        if item.created_on > (
+                item.last_updated + datetime.timedelta(seconds=2)):
             cls._add_error(
                 ERROR_CATEGORY_TIME_FIELD_CHECK,
                 'Entity id %s: The created_on field has a value %s which '
                 'is greater than the value %s of last_updated field'
                 % (item.id, item.created_on, item.last_updated))
 
-        current_datetime = datetime.datetime.utcnow()
+        current_datetime = datetime.datetime.utcnow() + datetime.timedelta(
```

Comment:
I don't think you need the tolerance here ... do you?

Reply:
Yes, verified no such error occured. Removed.